include/object/registry.hpp

```diff
@@ -45,6 +45,8 @@ namespace ntw::obj {
                 auto  upath = make_ustr(path);
                 auto  attr  = make_attributes(&upath, OBJ_CASE_INSENSITIVE);
                 ULONG disposition;
+
+                _handle.reset();
```

Comment:
Again. I am fine with these changes but with extra effort we could guarantee that if opening fails we still have the old handle.